src/components/NewLocationPage/CompareLocationTabs/CompareLocationTabs.style.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import ToggleButton from '@material-ui/lab/ToggleButton';
+import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
+
+export const ButtonGroup = styled(ToggleButtonGroup)`
+  background-color: ${COLOR_MAP.GREY_1};
+  border: 1px solid ${COLOR_MAP.GREY_2};
+  border-radius: 4px;
+`;
+
+export const Button = styled(ToggleButton)`
```

Comment:
these buttons need some MUI-specific props to remove the ripple that occurs when clicking. you can see an example of how those props are applied [here in ShareButtons.style](https://github.com/covid-projections/covid-projections/blob/develop/src/components/ShareButtons/ShareButtons.style.tsx#L8)